sass/layout/_sidebar-left.scss

```diff
@@ -1075,7 +1075,6 @@
     }
 
     .SidebarChannel span.SidebarChannelLinkLabel {
-        width: 100%;
```

Comment:
Why remove this?

Reply:
@hmhealey Thank you for pointing this out. We were able to identify and fix a related issue.

The name width is changed from 100% to a max of 85% to support custom emojis next to the name. 

![image](https://user-images.githubusercontent.com/35728906/108058440-0f4da580-707a-11eb-9719-1e233293bdde.png)

Reply:
Should that be needed? I thought we size those items with flexbox so that would scale automatically. We size it dynamically with the mention badge, but maybe that's done manually instead of with flexbox. Right now, long channel names are cut off early even if they're not DMs or if they don't have the status icon.

Test server:
![Screen Shot 2021-02-16 at 2 40 27 PM](https://user-images.githubusercontent.com/3277310/108113043-f25aa780-7064-11eb-83ba-c0d27f8081e8.png)
My server:
![Screen Shot 2021-02-16 at 2 35 57 PM](https://user-images.githubusercontent.com/3277310/108113053-f71f5b80-7064-11eb-9f63-2ff895386056.png)